modules/electromagnetics/doc/content/source/kernels/ConductivityLaplacian.md

```diff
@@ -0,0 +1,16 @@
+# ConductivityLaplacian
```

Comment:
> there are conductivities in the same shape of equation in other physics

Yes - this was named in a situation for Freya where I wanted the name to be very clear as to what it was being used for, and was completely dissatisfied with the MOOSE naming convention. But I admit it could be replaced by `ADMatDiffusion`. Will do.